pom.xml

```diff
@@ -2134,12 +2122,29 @@
                <configuration>
                   <autoReleaseAfterClose>true</autoReleaseAfterClose>
                   <stagingDescription>${infinispan.brand.name} ${project.version} release</stagingDescription>
-                  <nexusUrl>${jboss.releases.nexus.url}</nexusUrl>
-                  <serverId>${jboss.releases.repo.id}</serverId>
-                  <stagingProfileId>2161b7b8da0080</stagingProfileId>
+                  <nexusUrl>${maven.releases.nexus.url}</nexusUrl>
+                  <serverId>${maven.releases.repo.id}</serverId>
                   <stagingRepositoryId>${stagingRepositoryId}</stagingRepositoryId>
                </configuration>
             </plugin>
+            <plugin>
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-gpg-plugin</artifactId>
+               <version>${version.maven.gpg}</version>
+               <executions>
+                  <execution>
+                     <id>sign-artifacts</id>
+                     <phase>verify</phase>
+                     <goals>
+                        <goal>sign</goal>
+                     </goals>
+                     <configuration>
+                        <keyname>9E31AB27445478DB</keyname>
+                        <passphraseServerId>9E31AB27445478DB</passphraseServerId>
```

Comment:
shouldn't these be properties and be set in Jenkins `settings.xml`?

Reply:
The key name is public: https://keyserver.ubuntu.com/pks/lookup?search=9E31AB27445478DB&fingerprint=on&op=index

Reply:
but if you want to run/sign locally, you need to change the hardcoded values :)

Reply:
I don't understand the need to sign artifacts with the non-official key

Reply:
fair enough. :+1: